index.js

```diff
@@ -219,4 +219,33 @@ myForm.addEventListener("submit", (event) => {
         errorMessage.classList.remove("d-none");
         event.preventDefault();
     }
-});
\ No newline at end of file
+});
+
+let inputs = Array.from(document.getElementsByClassName("input"));
+
+let formInputValues = {
+    firstname:"",
+    lastname:"",
+    email: "",
+    message: "",
+};
+
+if (!localStorage.getItem('form')) {
+    localStorage.setItem('form', JSON.stringify(formInputValues));
```

Comment:
The form data for all inputs should be stored in a single object; saved to a single local storage register 🙌